strapi/strapi
# chore: update commander to 11.1.0

### What does it do?

- updates commander from 8.3.0 -> 11.1.0

[There have been breaking changes](https://github.com/tj/commander.js/blob/master/CHANGELOG.md) but they do not appear to affect our usage -- however, this should be thoroughly tested and if anything is found to be broken I can fix and add a test for it.

### Why is it needed?

We're several versions behind and are taking the opportunity to update in v5

### How to test it?

Test all the CLI commands.

There was a breaking change regarding default boolean properties that seem fine, but it's possible something broke so please confirm.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
